pkgs/development/web/nodejs/nodejs.nix

```
@@ -102,9 +102,14 @@ let
       "--with-arm-float-abi=${gcc.float-abi}"
     ]) ++ extraConfigFlags;
 
-    configurePlatforms = [];
+    configurePlatforms = [ ];
 
-    dontDisableStatic = true;
+    configurePhase = ''
+      runHook preConfigure
+      CC_host="$CC_FOR_BUILD" CXX_host="$CXX_FOR_BUILD" \
+        ${python.executable} configure.py --prefix="$prefix" "''${configureFlags[@]}"
```

I'm not a bash expert, but whats that `''${configureFlags[@]}` about ?
```suggestion
        ${python.executable} configure.py --prefix="$prefix" "${configureFlags[@]}"
```

Shouldn't it work without these empty quotes ?

---

That’s Nix, not Bash 😅
Double quotes escape interpolation in multiline string, otherwise `configureFlags[@]` would be interpreted as a Nix expression.

---

Hm, I wanted to post a link to the documentation but it looks like it’s not currently documented though IIRC it was before…
https://nix.dev/manual/nix/2.18/language/string-interpolation

Anyway, https://releases.nixos.org/nix-dev/2015-December/019018.html

---

Oh right. The weird escaping of `$`

`{s = '' ''${} ''; }`